fuse_core/include/fuse_core/transaction_deserializer.h

```diff
@@ -89,6 +89,7 @@ class TransactionDeserializer
 private:
   pluginlib::ClassLoader<fuse_core::Variable> variable_loader_;  //!< Pluginlib class loader for Variable types
   pluginlib::ClassLoader<fuse_core::Constraint> constraint_loader_;  //!< Pluginlib class loader for Constraint types
+  pluginlib::ClassLoader<fuse_core::Constraint> loss_loader_;  //!< Pluginlib class loader for Loss types
```

Comment:
Again, assuming this should be:
```suggestion
  pluginlib::ClassLoader<fuse_core::Loss> loss_loader_;  //!< Pluginlib class loader for Loss types
```

Reply:
Fixed.

When I tested it with `fuse_echo` it was printing the loss function values properly, so maybe this wasn't really needed. Either way, I think we can look at whether all these loaders are needed or not later.